net.gaijin.warthunder.yaml

```
@@ -0,0 +1,31 @@
+app-id: net.gaijin.warthunder
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
+  - --device=dri
+  # =all for controller support
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+#  - --filesystem=~/.config/WarThunder/
+#  - --persist=.config
+
+modules:
+  - name: warthunder
+    buildsystem: simple
+    build-commands: 
+      - mkdir /app/bin/
+      - install -D ./run.sh /app/bin/
```

```suggestion
      - install -D -t ${FLATPAK_DEST}/bin/ ./run.sh 
```

otherwise it doesn't work (dest is a filename unless using -t, and -D will create dirs as needed)

---

-D creates directories

---

you confuse with `-d`.

```
 -D                  create all leading components of DEST except the last,
                        or all components of --target-directory,
                        then copy SOURCE to DEST
```

---

If I am confusing it, Then why does it run perfectly?

---

if you remove the mkdir your command will say 
```
install: target '/app/bin/' is not a directory: No such file or directory
```